app/vmctl/flags.go

```diff
@@ -553,6 +554,11 @@ var (
 			Usage: "Whether to skip TLS certificate verification when connecting to the remote read address",
 			Value: false,
 		},
+		&cli.BoolFlag{
+			Name:  remoteReadDisablePathAppend,
+			Usage: "Whether to disable automatic appending of the path to the remote storage.",
+			Value: true,
```

Comment:
Why the default value of this flag is set to true? It looks like it will break backwards compatibility, since the previous versions of `vmctl` assume that this flag is set to false.

Reply:
I fixed it in the follow-up commit b50ed5ddd12c43f1978b22e9de0d855bdd9efc03 . The `Migrating data from Promscale` chapter must be added though in a separate pull request.